Jenkinsfile

```diff
@@ -1,7 +1,6 @@
 pipeline {
     agent {
         docker {
-            alwaysPull true
```

Comment:
Please revert this change.  It was a temporary attempt to get around the OCIR outage.